client/service/workspace_configrations.go

```diff
@@ -0,0 +1,39 @@
+package service
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// WorkspaceConfAPI exposes the workspace configurations API
+type WorkspaceConfAPI struct {
```

Comment:
Move this file to `workspace` package, because it's making configuration of workspace